A Box2D dependent GWT module fails to build

#### Issue details

After upgrading my projects to LibGDX 1.9.11 (from 1.9.10) the GWT modules fail to build if they include Box2D dependencies.

This is the error message from `./gradlew html:superDev`

```
Loading inherited module 'com.mygdx.game.GdxDefinitionSuperdev'
   Loading inherited module 'com.badlogic.gdx.physics.box2d.box2d-gwt'
      [ERROR] Unable to find 'com/badlogic/gdx/physics/box2d/box2d-gwt.gwt.xml' on your classpath; could be a typo, or maybe you forgot to include a classpath entry for source?
```

Apparently the `box2d-gwt.gwt.xml` file is missing in [`com.badlogicgames.gdx:gdx-box2d-gwt:1.9.11:sources`](https://repo1.maven.org/maven2/com/badlogicgames/gdx/gdx-box2d-gwt/1.9.11/gdx-box2d-gwt-1.9.11-sources.jar)

This issue is most likely a regression from #5956

#### Reproduction steps/code
1. Generate a clean project using the setup app with HTML backend and Box2D dependency.
2. Issue `./gradlew html:superDev`

#### Version of LibGDX and/or relevant dependencies
- 1.9.11
- Box2D
- GWT

#### Please select the affected platforms
- [x] HTML/GWT
- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [ ] Windows
- [ ] Linux
- [ ] MacOS

--- a/extensions/gdx-box2d/gdx-box2d-gwt/pom.xml
+++ b/extensions/gdx-box2d/gdx-box2d-gwt/pom.xml
@@ -18,13 +18,13 @@
 			<artifactId>gdx</artifactId>
 			<version>${project.version}</version>
 		</dependency>
-		
+
 		<dependency>
 			<groupId>${project.groupId}</groupId>
 			<artifactId>gdx-box2d</artifactId>
 			<version>${project.version}</version>
 		</dependency>
-		
+
 		<dependency>
 			<groupId>com.google.gwt</groupId>
 			<artifactId>gwt-user</artifactId>
@@ -54,7 +54,12 @@
 						</goals>
 					</execution>
 				</executions>
-			</plugin>            
+			</plugin>
 		</plugins>
+		<resources>
+			<resource>
+				<directory>res</directory>
+			</resource>
+		</resources>
 	</build>
 </project>

